feedly/api_client/utils.py

```diff
@@ -3,9 +3,8 @@
 from typing import Callable
```

Comment:
can this file live in the examples folder? it's not useful in the library

Reply:
imports won't work if it's in the examples folder since we moved it out of the lib